serverlessworkflow/modules/ROOT/pages/cloud/operator/enabling-jobs-service.adoc

```diff
@@ -0,0 +1,170 @@
+= Managing Jobs Service with the Operator
+:compat-mode!:
+// Metadata:
+:description: Configure Jobs Service using the `SonataFlowPlatform` CR.
+:keywords: sonataflow, serverless, operator, kubernetes, jobs service
+
+
+This document describes how to configure the Jobs Service instance usimg the SonataFlowPlarform CR.
+
+== Automate the Jobs Service instance management with the `SonataFlow` Operator
+
+While it is possible to deploy the Jobs Service manually, the Operator also provides a convenient way to combine it with the
+namespace configuration via the `SonataFlowPlatform` CR. With this approach, the Operator will take care of configuring the
+Jobs Service instance and ensure it is in sync with the specification in the CR. When the operator manages the Jobs Service lifecycle,
+it will inject properties in `SonataFlow` workflows at creation time to enable the workflows to communicate with the Jobs Service
+instance during their execution, removing the need to add these properties as part of the `SonataFlow` workflow CR instance.
+
+== Configuring Jobs Service in the SonataFlowPlatformCR
+
+To enable the deployment of a Jobs Service instance, the `SonataFlowPlatform` CRD exposes a set of fields that allow the user to 
+configure the running instance. 
+
+==== Ephemeral persistence
+The basic runtime is to deploy the Jobs Service with an ephemeral backend running in the same container
+as the Jobs Service runtime.
+
+[source,yaml,subs="attributes+"]
+---
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlowPlatform
+metadata:
+  name: sonataflow-platform
+spec:
+  services:
+    jobService: {}
+---
+
+When executing this manifest, the operator will reconcile generating a pod hosting the Jobs Service:
+
+[source,shell,subs="attributes+"]
+---
+$>kubectl get pod -n sonataflow
+NAME                                               READY   STATUS    RESTARTS   AGE
+sonataflow-platform-jobs-service-cdf85d969-sbwkj   1/1     Running   0          108s
+---
+
+Keep in mind that this setup is not recommended for production environments, chiefly because the data is not persisted when the pod restarts.
+
+==== Using an existing postgreSQL service
+For robust environments it is recommened to use an dedicated database service and configure Jobs Service to make use of it. Currently, the Jobs Service
+only supports PostgreSQL database.
+
+Configuring Jobs Service to communicate with an existing PostgreSQL instance is supported in two ways. In both cases it requires providing the persistence
```

Comment:
```suggestion
Configuring Jobs Service to communicate with an existing PostgreSQL instance is supported in two ways. In both the cases it requires providing the persistence
```

Reply:
Not sure if this sounds right, should the suggestion be `In both of the cases`?

Reply:
According to [this](https://forum.thefreedictionary.com/postst216720_Both--Both-the.aspx) website, the usage of "both the <noun>" is relevant when the `noun` is singular:

> Both the sentences are grammatically correct. "The" is usually not necessary when the ensuing noun is plural and only one noun is used e.g. both sisters, both pictures and both children.
> 
> "The" is required in the following pattern in singular: Both the elephant and the bison, both the first and the last book, both the boy and the girl...
> 
> Additionally, "the" is required in the following pattern: Both the sisters and the brothers. You do not say: Both sisters and the brothers.

Disclaimer: I'm not a native speaker